src/client/app/utils/hasPermissions.ts

```diff
@@ -5,10 +5,11 @@
 import { UserRole } from '../types/items';
 
 /**
- * Checks if the user has the permissions of a given role.
- * @param {UserRole} user User role to evaluate
- * @param {UserRole} compareTo User role to compare to
- * @returns {boolean} Whether or not the user has the compareTo role
+ * Checks if the role A has the permission of the role B.
+ *
+ * @param {UserRole} userA variable for role A
```

Comment:
I like these changes but could we change:
variable for role A/B
to
role for A/B
I suggest this since, in general, we don't put the word "variable" in an @param since most describe variables.

Reply:
Hello @huss . Sorry for the late reply. Since @Bedolla233 didn't modify it based your comment, I, as his teammate, helped him modify it. Thank you for your patience and comments.

Reply:
I updated the PR a little. This was mostly due to another PR also changing the same file/comments. The PR is now fine/done from what you did. I am currently seeing if I can get it to pass the GitHub action testing as it started to fail. This has nothing to do with the changes you made.